Добавление классов

Maximmutylin opened the pull request:
Описание первых шагов и добавление классов с логикой игры

Maximmutylin committed:
Добавление классов

Описание первых шагов и добавление классов с логикой игры

KazaninaYana commented on index.html:
```diff
@@ -0,0 +1,33 @@
+            <div class="cell"></div>
+            <div class="cell"></div>
+            <div class="cell"></div>
+            <div class="cell"></div>
```
Все 16 ячеек нужно сгенерировать из кода, а не прописывать вручную)
Создать 16 раз див внутри блока с классом grid)

KazaninaYana commented on script.js:
```diff
@@ -0,0 +1,110 @@
+    init() {
+        this.board = new Board()
+        this.board.init()
+		document.addEventListener('keyup', clickControl);
```
отступ съехал

Maximmutylin committed:
Перенос функций  addColours и generateNewCell

addColours - добавляет цвет ячейки при появлении в зависимости от ее значения.
generateNewCell - генерирует первое значение в рандомной клетке поля.
Исправил ручное добавление cell

KazaninaYana commented on script.js:
```diff
@@ -0,0 +1,122 @@
+'use strcit'
```
Опечатка, должно быть 'use strict'
Не пропускай точки с запятой, в проекте много где их нет)

Maximmutylin committed:
feat::  Placed classes in modules

Maximmutylin committed:
fix: fix syntax errors

KazaninaYana commented on classBoard.js:
```diff
@@ -0,0 +1,38 @@
+'use strict'
+
```
Не нужно в названии класса прописывать, что это класс, это по сути же никому не интересно) Какая там реализация)

KazaninaYana commented on classBoard.js:
```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+import { colorCell } from './script.js'
```
не ставишь ; здесь и в 1 строке,

KazaninaYana commented on classBoard.js:
```diff
@@ -0,0 +1,38 @@
+    
+    generateNewCell() {
+        const randomNumber = Math.floor(Math.random() * this.squares.length);
+        if (this.squares[randomNumber].innerHTML === '') {
```
между объявлением переменных и if стоит добавить пустую строку

KazaninaYana commented on script.js:
```diff
@@ -0,0 +1,60 @@
+        console.log('Вправо') ;  
+    }else if(event.key === 'ArrowLeft') {
+        console.log('Влево');
+    } else {
```
Здесь с пробелом } else { а выше пробелов после первой фигурной скобки нет

KazaninaYana commented on script.js:
```diff
@@ -0,0 +1,60 @@
+}
+
+import { Cell } from './classCell.js'
+import { Board } from './classBoard.js'
```
Импорты принято объявлять вверху страницы, самыми первыми строками

KazaninaYana commented on script.js:
```diff
@@ -0,0 +1,60 @@
+    }
+}
+
+import { Cell } from './classCell.js'
```
Здесь Cell и не нужен) В том числе и this.test = new Cell();
Он нужен будет в классе Board

KazaninaYana commented on style.css:
```diff
@@ -0,0 +1,96 @@
+    line-height: 15vmin;
+    color: #776e65;
+    transition: all .3s;
+}
```
Смотри, сейчас нужно исправить моменты, которые я здесь описала, запушить в эту же ветку и отправить еще раз на проверку, затем, когда всё будет исправлено - залить в мейн правки и только после этого переходить к новому заданию)

Maximmutylin committed:
fix: syntax

Maximmutylin committed:
fix: indentation and filenames
